flock_test.go

```diff
@@ -45,36 +45,30 @@ func (t *TestSuite) TearDownTest(c *C) {
 }
 
 func (t *TestSuite) TestNew(c *C) {
-	var f *flock.Flock
-
-	f = flock.New(t.path)
+	f := flock.New(t.path)
 	c.Assert(f, Not(IsNil))
 	c.Check(f.Path(), Equals, t.path)
 	c.Check(f.Locked(), Equals, false)
 	c.Check(f.RLocked(), Equals, false)
 }
 
 func (t *TestSuite) TestFlock_Path(c *C) {
-	var path string
-	path = t.flock.Path()
+	path := t.flock.Path()
 	c.Check(path, Equals, t.path)
 }
 
 func (t *TestSuite) TestFlock_Locked(c *C) {
-	var locked bool
```

Comment:
To avoid a bunch of the same kinds of comments, please revert the changes where the compile time type check is removed.